ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml

```diff
@@ -97,6 +97,58 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-vsphere-ipi-ovn
   timeout: 5h0m0s
+- always_run: false
+  as: vcenter-1-control-plane-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: vsphere-elastic
+    env:
+      DEFAULT_NETWORK_TYPE: public-ipv6
+      POOLS: vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-vsphere-ipi-multi-vcenter
+  timeout: 5h0m0s
+- always_run: false
+  as: vcenter-1-control-plane-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: vsphere-elastic
+    env:
+      DEFAULT_NETWORK_TYPE: public-ipv6
+      POOLS: vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
+      WORKER_REPLICA_COUNT: "24"
+    test:
+    - chain: workers-scale
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-vsphere-ipi-multi-vcenter
+  timeout: 5h0m0s
+- always_run: false
+  as: vcenter-1-ipv4ipv6dual-control-plane-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: vsphere-elastic
+    env:
+      DEFAULT_NETWORK_TYPE: public-ipv6
+      POOLS: vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-vsphere-ipi-dualstack
+  timeout: 5h0m0s
+- as: vcenter-1-ipv4ipv6dual-control-plane-24nodes
+  cron: 0 7 2,9,16,23 * *
```

Comment:
is this kinda excessive? are you notifying the users of vcenter-1 devqe for each run?

Reply:
@jcpowermac 
> is this kinda excessive?
Once a week is not very frequent for CI. We will have another job in the future, at that time, I can make this twice a month and the new one twice a month.

> are you notifying the users of vcenter-1 devqe for each run?

No. I'm not aware of the rest users of vcenter-1 devqe. When I requested this env Jim said "we could set up a new vCenter to manage that 2 node cluster and configure it for ipv6. This would also not impact the current DEVQE set up and resources (which are over half for cpu and memory for the cluster).". So I supposed the rest users are on the previous envs. https://issues.redhat.com/browse/DPP-14784?focusedId=24737745&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24737745. 
I can see some other vms on the vcenter console, 3 of them have pewang in the vm names, I have noticed him. But I can not tell who is the owner for the rest of them, like the vms with prefix 'nested7' and 'nested8'. Would you suggest what I should do?